massa-versioning/src/versioning.rs

```diff
@@ -1266,29 +1266,41 @@ mod test {
         }
     }
 
-    fn get_a_version_info() -> (NaiveDateTime, NaiveDateTime, MipInfo) {
+    fn get_a_version_info() -> (OffsetDateTime, OffsetDateTime, MipInfo) {
         // A helper function to provide a default MipInfo
+
         // Models a Massa Improvements Proposal (MIP-0002), transitioning component address to v2
 
-        let start: NaiveDateTime = NaiveDate::from_ymd_opt(2017, 11, 01)
+        let start_datetime = Date::from_calendar_date(2017, Month::November, 01)
             .unwrap()
-            .and_hms_opt(7, 33, 44)
-            .unwrap();
-
-        let timeout: NaiveDateTime = NaiveDate::from_ymd_opt(2017, 11, 11)
+            .with_hms(7, 33, 44)
             .unwrap()
-            .and_hms_opt(7, 33, 44)
-            .unwrap();
+            .assume_utc();
+        let timeout_datetime = Date::from_calendar_date(2017, Month::November, 11)
+            .unwrap()
+            .with_hms(7, 33, 44)
+            .unwrap()
+            .assume_utc();
 
         return (
-            start,
-            timeout,
+            start_datetime,
+            timeout_datetime,
             MipInfo {
                 name: "MIP-0002".to_string(),
                 version: 2,
                 components: BTreeMap::from([(MipComponent::Address, 1)]),
-                start: MassaTime::from_millis(start.timestamp() as u64),
-                timeout: MassaTime::from_millis(timeout.timestamp() as u64),
+                start: MassaTime::from_millis(
+                    start_datetime
```

Comment:
we should use MassaTime everywhere. Why are we using NaiveDateTime and OffsetDateTime at all ?

Reply:
Changed to MassaTime in f0c9d7ee374c4494387b61e513110fb520942a86